todo/src/components/Todo.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import { addTodo, crossOut } from '../actions'
+import TodoForm from './TodoForm'
+class Todo extends React.Component {
+  constructor(props) {
+    super(props)
+      this.state = {
+        inputText: '',
+      }
+  }
+
+  changeHandler = (e) => {
+    this.setState({inputText: e.target.value})
+  }
+
+  handleNewItem = (e) => {
+    e.preventDefault()
+    console.log('button clicked')
+    this.props.addTodo(this.state.inputText)
+    this.setState({inputText: ''})
+  }
+
+  crossOut = (e, id) => {
+    e.preventDefault();
+    this.props.crossOut(id)
+  }
+
+  render() {
+    return (
+      <div>
+      <TodoForm
```

Comment:
You have two duplicate forms. The difference is that one is not working properly (wrong props are beign passed) so to not render both of them it is suggested to comment out/remove that one for time beign.